api/metrics.go

```diff
@@ -119,19 +119,24 @@ func recordGauges(c *gin.Context) {
 		logrus.Errorf("Error while reading all error builds: %v", err)
 	}
 
-	stepStatusMap, err := database.FromContext(c).GetStepStatusCount()
+	stepImageMap, err := database.FromContext(c).GetStepImageCount()
 	if err != nil {
-		logrus.Errorf("Error while reading all error builds: %v", err)
+		logrus.Errorf("Error while reading all step images: %v", err)
 	}
 
-	stepImageMap, err := database.FromContext(c).GetStepImageCount()
+	stepStatusMap, err := database.FromContext(c).GetStepStatusCount()
 	if err != nil {
-		logrus.Errorf("Error while reading all images: %v", err)
+		logrus.Errorf("Error while reading all step statuses: %v", err)
```

Comment:
I'm guessing this is similar error handling to the others, but does it make sense to fix them now?

I believe we do not capitalize the first word and went are from stating error in an error because it's redundant.

Reply:
Yeah I thought about fixing them but wasn't sure if it made sense to do it in this PR due to scope 🤷 

I'll just update them to be safe 👍

Reply:
Thanks!

Reply:
@kneal fixed! 🎉